README.md

```diff
@@ -11,48 +11,6 @@
 5. (Windows users only) Install required package, `conda install fiona`
 7. Install development version of package, `pip install -e .`
 
-### Database setup
-
-Our production database is an instance of PostgreSQL on google cloud SQL.
-
-All of our SQL setup and interactions happen through sqlalchemy, which is
-(mostly) database engine agnostic.
-
-For running integration tests locally there are two options:
-
-1. Use an in-memory sqlite backend: This is the default setup and will happen
-   automatically for you when you run `pytest` operations from the local
-   directory (see training/training.org for more info on pytest options).
-2. Use a PostgreSQL server: To use a PostgreSQL server instead, you must set the
-   environment variable `CAN_PG_CONN_STR` to a proper PostgreSQL connection URI
-   before running pytest. Again see training/training.org for more info
-
-
-If you would like to work interactively in an IPython session or Jupyter
-notebook, you can use the function `can_tools.models.create_dev_engine` to set
-up an in-memory SQLite instance
-
-Below is a code snippet that sets this up, and then runs the `Florida` scraper
-and inserts data into the database
-
-```python
-from can_tools.models import create_dev_engine
-from can_tools.scrapers import Florida
-
-# setup databsae
-engine, Session = create_dev_engine()
-
-scraper = Florida()
-df = scraper.normalize(scraper.fetch())
-
-scraper.put(engine, df)
-```
-
-Note that by default the `create_dev_engine` routine will construct the database
-in a verbose mode where all SQL commands are echoed to the console. We find that
-this is helpful while debugging and developing. This can be disabled by passing
-`verbose=False` when calling `create_dev_engine`.
-
```

Comment:
Hey @jrybacek was removing this section of the docs intentional?

I believe it is still accurate. If not, please let me know.

Reply:
Hmm, yeah.  Its still accurate.  At the time I read it, I was thinking it was extraneous, but it does make sense to keep for anyone hoping to run a local PostgreSQL server.  I'll revert this change.